modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -1,21 +1,99 @@
+
+Your applications talk to Serverless over the public internet using encrypted connections. Serverless does not support private networking (that is, VPC peering to talk over private IPs). Use Dedicated clusters for private networking.
+
+Available on AWS.
+
+=== Dedicated Cloud
+
+With Dedicated clusters, you host your data in Redpanda's VPC, and Redpanda handles provisioning, operations, and maintenance. Dedicated clusters are single-tenant deployments for better data isolation. 
+
+Available on AWS or GCP. 
+
+=== Bring Your Own Cloud (BYOC)
+
+With BYOC clusters, you deploy Redpanda in your own VPC, so all data is
+contained in your own environment. There are no inbound connections from the Redpanda
+control plane to the data plane. This provides an additional layer of security
+and isolation. Redpanda handles provisioning, operations, and maintenance. 
+
+Setting up a BYOC cluster is initiated from the Redpanda control plane. This is a two-step process performed using `rpk`:
+
+. You bootstrap a virtual machine (VM) in your VPC. This VM spins up the agent
+and the required infrastructure.
+. The agent communicates with the control plane to pull the cluster specifications.
+
+When provisioning BYOC, at the time an agent is deployed, Redpanda assigns the
+necessary IAM policies required to run the agent.
+
+NOTE: With standard BYOC clusters, Redpanda manages security policies and resources for your VPC, including subnetworks, service accounts, IAM roles, firewall rules, and storage buckets. For the most security, you can manage these resources yourself with a xref:./vpc-byo-gcp.adoc[customer-managed VPC].
+
+Available on AWS or GCP. 
+
+== Serverless vs Dedicated/BYOC
+
+Serverless clusters are a good fit for the following:
+
+* Starter and growing workloads
+* Spiky workloads (development environments, systems that only occasionally get busy, or workloads that come and go), only paying for what you consume, without any commitment
+* Fast and dynamic cluster creation: you can use a Serverless cluster as an isolated container for topics
+
+Because a Serverless cluster is created in under one second, you can surface it in your applications (for example, for tenant isolation). As your workload increases, you can can migrate a Serverless cluster to a Dedicated or BYOC cluster.
+
+Dedicated and BYOC clusters offers the following features:
+
+* Single-zone or multi-zone availability (a multi-zone cluster provides higher resiliency in the event of a failure in one of the zones)
+* Private networking using VPC peering
+* Ability to export metrics to a 3rd-party monitoring system
+* Managed connectors
+* Unlimited retention with glossterm:Tiered Storage[]
+
+Consider Dedicated or BYOC if you need any of the following:
```

Comment:
Would it be good to mention here that Dedicated or BYOC are necessary if you're using GCP?

Reply:
GCP for serverless is coming soon.